bibtex/working-group.bib

```diff
@@ -0,0 +1,601 @@
+}
+
+@unpublished{steeves_reproducing_2017,
+	address = {Lawrence, Kansas},
+	title = {Reproducing and {Preserving} {Research} with {ReproZip}},
+	copyright = {CC-By Attribution 4.0 International},
+	url = {https://vickysteeves.gitlab.io/2017-IASSIST-ReproZip/#/},
+	author = {Steeves, Vicky and Rampin, Rémi},
+	month = {May},
+	note = {IASSIST 2017},
+	year = {2017}
+}
+
+@unpublished{steeves_creating_2017,
+	address = {Austin, Texas},
+	title = {Creating {Reproducible} {Experiments} with {ReproZip}},
+	copyright = {CC-By Attribution 4.0 International},
+	url = {https://vickysteeves.gitlab.io/2017-SciPy/#/},
+	abstract = {Reproducibility is a core component of the scientific process: it helps researchers all around the world to verify research results and also to build on them, allowing science to move forward. Unfortunately, computational reproducibility can be very painful. We’ll present an open source tool for computational reproducibility, ReproZip. ReproZip is written in Python, and was designed to simplify the process of making an experiment reproducible across platforms. ReproZip creates self-contained, reproducible packages by automatically tracking, identifying, and capturing all its required dependencies: programs, libraries, data, and configuration files. The original user can share the package with others, who can then use ReproZip to unpack and rerun the experiment on their favorite operating system.},
+	language = {en},
+	note = {SciPy},
+	author = {Steeves, Vicky and Rampin, Rémi},
+	month = {July},
+	year = {2017}
+}
+
+@unpublished{steeves_organizational_2015,
+	address = {Washington, D.C},
+	title = {Organizational {Implications} of {Data} {Science} {Environments} in {Education}, {Research}, and {Research} {Management} in {Libraries}},
+	url = {https://osf.io/rhgq3/},
+	abstract = {A presentation by Vicky Steeves, Erik Mitchell, and Jennifer Muilenburg on the implications of data science in a library environment.},
+	language = {en},
+	note = {CNI Fall Meeting},
+	author = {Steeves, Vicky and Mitchell, Erik and Muilenburg, Jennifer},
+	month = {December},
+	year = {2015}
+}
+
+@misc{group_reproducible_2017,
+	title = {Reproducible {Science} @ {NYU}},
+	url = {https://nyu.reproduciblescience.org},
+	abstract = {A website that gives an overview to the work of the NYU Open Science and Reproducibility Working Group.},
+	language = {en},
+	journal = {Reproducible Science @ NYU},
+	author = {Group, NYU Reproducibility Working},
+	month = {January},
+	year = {2017}
+}
+
+@misc{group_reproducible_2018,
+	title = {Reproducible {Science}},
+	url = {https://reproduciblescience.org/},
+	abstract = {On this site, the moderators have curated sources of various types discussing reproducibility in the directory. You can find academic papers, blog posts, popular media articles, talks, tools, and more.},
+	language = {en},
+	journal = {Reproducible Science},
+	author = {Group, NYU Reproducibility Working},
+	month = {June},
+	year = {2018}
```

Comment:
Should this also be January 2017? Or August 2015, since this is when you set up the wordpress version.